packages/commonwealth/client/scripts/controllers/chain/substrate/collective_proposal.ts

```diff
@@ -120,22 +120,9 @@ export class SubstrateCollectiveProposal
 
     entity.chainEvents.forEach((e) => this.update(e));
 
-    if (!this._completed) {
-      const slug = chainEntityTypeToProposalSlug(entity.type);
-      const uniqueId = `${slug}_${entity.typeId}`;
-      this._Chain.app.chain.chainEntities._fetchTitle(entity.chain, uniqueId).then((response) => {
```

Comment:
How so? This was only because incomplete events were fetched from the chain so we did not have the associated titles but now ALL entities are loaded from the database so the title is inherently included.